src/main/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedCronTab.java

```diff
@@ -22,22 +19,14 @@
 public class ParameterizedCronTab {
 
 	private final Map<String, String> parameterValues;
-	private final long[] bits;
-	private final Integer dayOfWeek;
-	private final CronTab cronTab;
```

Comment:
This class is not persisted on the disk: https://github.com/jenkinsci/parameterized-scheduler-plugin/blob/4a47537d58f4f5a0b528ba710a4f849bcf3459bf/src/main/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedTimerTrigger.java#L30 , so such cleanup should be fine